virttest/utils_split_daemons.py

```diff
@@ -0,0 +1,394 @@
+"""
+Module to control split daemon service.
+"""
+# pylint: disable=E0611
+import re
+import logging
+
+import aexpect
+from avocado.utils import process
+from avocado.utils import wait
+
+from virttest import libvirtd_decorator
+from virttest import remote
+from virttest import utils_misc
+from virttest.staging import service
+from virttest.utils_gdb import GDB
+
+
+class VirtDaemonCommon(object):
+
+    """
+    Common class to manage libvirt split daemon service on host or guest.
+    """
+    daemon_name = ''
+    virsh_cmd = "virsh -c qemu:///system list"
```

Comment:
why not put them all in __init__?

Reply:
some services are different, such as networkd is virsh ="virsh -c network://system list"

Reply:
this is difference between Class Attribute vs. Instance Attribute.
put virsh_cmd in class attribute to avoid additional constructor parameter and differeitated passing in value for constructor function

Reply:
Sorry I don't understand. Even if you use the class attr, it is only valuable when you using class.attr, and you can also get/change the value with inst.attr/self.attr in instance usage(Although class.attr not changed but you do not use it).

Reply:
Or do you mean you will change it later like using for non root "session" ?

Reply:
this is similar usage with daemon_name. At  least one benefit is that virsh_cmd can avoid to be pass in by constructor. If put virsh_cmd as one parameter in constructor, that means when you construct virtqemud or virtproxyd, you need virtproxyd = virtproxyd(' virsh_cmd' ,xx,xx)